rtos/TARGET_CORTEX/rtx4/cmsis_os.h

```diff
@@ -438,23 +438,23 @@ uint32_t osKernelSysTick (void);
 /// \param         instances     number of possible thread instances.
 /// \param         stacksz       stack size (in bytes) requirements for the thread function.
 #if defined (osObjectsExternal)  // object is external
-#define osThreadDef(name, priority, instances, stacksz) \
+#define osThreadDef(name, priority, stacksz) \
```

Comment:
Could you remove the instances parameter from the function documentation ?

Reply:
Fixed